packages/web-react/src/components/Modal/README.md

```diff
@@ -0,0 +1,55 @@
+# Modal
+
+```jsx
+import { Modal, ModalHeader, ModalBody, ModalFooter } from '@lmc-eu/spirit-web-react/components';
+```
+
+```jsx
+<Modal isOpen={isOpen} onClose={closeHandler}>
+  <ModalHeader>title</ModalHeader>
+  <ModalBody>body</ModalBody>
+  <ModalFooter>footer</ModalFooter>
+</Modal>
+```
+
+## SpiritModalProps
```

Comment:
Please see a heading level structure in `Tabs` for instance. We are using component name for heading and then `**Available props**`. We should be consistent.